### PR TITLE
Some 16bit Support helpter functions

### PR DESCRIPTION
**Pull request by goldhoorn**

### CODE REVIEW

**Comment by goldhoorn** [comment]
@D-Alex comments, otherwise i would like to merge

**Comment by goldhoorn** [comment]
_updated_

**Comment by goldhoorn** [comment]
_updated_

**Comment by goldhoorn** [comment]
_updated_
 i added a logarithmic scale

**Comment by goldhoorn** [comment]
now ok?

File: src/FrameQImageConverter.h [hunk 1]
```diff
@@ -66,6 +68,13 @@ namespace frame_helper
             //{
             int copyFrameToQImageRGB888(QImage &dst,base::samples::frame::frame_mode_t mode,int pixel_size, unsigned int width,unsigned int height, const char* pbuffer, const size_t buffer_size=0)
             {
+                //Safty check to prevent error on empty images
+                if(width == 0 || height  == 0 || buffer_size == 0){
```
Comment (jakobs) [inline src/FrameQImageConverter.h]: inconsistent use of newline before {. It seems the rest of the file is a mess, too, but I guess for new commits, at least they should be consistent.

**Comment by goldhoorn** [comment]
_updated_

**Comment by goldhoorn** [comment]
@D-Alex: pong

**Comment by jakobs** [comment]
looks good to me.

**Pull request merged**